test/quantization/fx/test_quantize_fx.py

```diff
@@ -4171,8 +4171,8 @@ def forward(self, x):
         # one quantize_per_tensor for input
         # check exact counts of quantize and dequantize
         count_check = {
-            ns.call_function(torch.quantize_per_tensor) : 1,
-            ns.call_method('dequantize') : 1
+            ns.call_function(torch.quantize_per_tensor) : 2,
```

Comment:
Will a more modular test help? Currently it is not clear what is exactly being tested.

Reply:
oh, this is testing that we do not have extra quant/dequant for all the ops listed in the forward function, these ops are what we call "CopyNode" in the quantize handler, this test might not be very useful in the future